### PR TITLE
fix: Lost loading collection's updateTs after qc restart

### PR DESCRIPTION
**Pull request by weiliu1031**
issue: #37537

### CODE REVIEW

**Comment by weiliu1031** [comment]
rerun go-sdk

File: internal/querycoordv2/meta/collection_manager.go [hunk 1]
```diff
@@ -169,8 +169,11 @@ func (m *CollectionManager) Recover(broker Broker) error {
 			}
 		}
 
+		// update collection's CreateAt and UpdateAt to now after qc restart
 		m.collections[collection.CollectionID] = &Collection{
 			CollectionLoadInfo: collection,
+			CreatedAt:          time.Now(),
```
Comment (xiaofan-luan) [inline internal/querycoordv2/meta/collection_manager.go:175]: partition may has the same issue?

**Comment by weiliu1031** [comment]
rerun cpp-unit-test

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**